src/Confluent.Kafka/Consumer.cs

```diff
@@ -27,22 +27,15 @@
 
 namespace Confluent.Kafka
 {
-    /// <summary>
-    ///     Implements a high-level Apache Kafka consumer (with 
-    ///     key and value deserialization).
-    /// </summary>
-    public class Consumer<TKey, TValue> : IDisposable
+    /// <inheritdoc cref="IConsumer{TKey,TValue}"/>
```

Comment:
Sorry, my mistake. I'm using Rider for development instead of Visual Studio. And Rider (and also ReSharper) are able to interpret those `<inheritdoc>` tags. If it's ok for you, i'll remove comments in the implementation and keep them only in the interface.